types/mapbox-gl/mapbox-gl-tests.ts

```diff
@@ -664,6 +665,12 @@ bounds.extend([
 // $ExpectType LngLatBounds
 bounds.extend([45, 30, 60, 60]);
 
+// controls
+// $ExpectAssignable IControl
```

Comment:
What is this $ExpectAssignable thing? This is the first time i see it. Cannot find any documentation on it.

Reply:
Could you try breaking it (removing "implements IControl" from "Control") and checking if the linter complains? I have some doubts here.

Reply:
You are right, this is now just a plain comment.

The new commit fixes this.